core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -110,6 +114,17 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
 </div>
 
 <style>
+  .modal-header {
+    background-color: white;
```

Comment:
```suggestion
    background-color: #fff;
```
Please use hex color codes.

Reply:
Thanks, this is now fixed